src/components/StudyDesignCriteria.vue

```diff
@@ -186,35 +186,40 @@
       </q-card-section>
 
     <!-- Height & Weight Range -->
-    <q-card-section>
+      <q-card-section
+        @input="
+          $emit('minbmi', v.minWeight.$model, v.minHeight.$model),
+          $emit('maxbmi', v.maxWeight.$model, v.maxHeight.$model)
+        ">
         <div class="row">
           <div class="col-4 q-pt-lg">
             <div class="text-bold">
               Height & Weight range:
             </div>
             <div class="text-caption">
               Minimum and maximum height/weight range of the participants.
+              BMI gets calculated automatically
             </div>
           </div>
           <div class="col q-pl-sm">
             <q-input
```

Comment:
I think it's better to use BMI directly instead of weight/height because the inclusion criteria may be directly expressed inn BMI. Please note that expressing height / weight constraints are not the same as expressing BMI constraints.